sagemaker-remote-function/pytorch_mnist_sample_notebook/pytorch_mnist.ipynb

```diff
@@ -0,0 +1,537 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #7.                        environment_variables={"AWS_DEFAULT_REGION": sm_session.boto_region_name}) as executor:
</pre><p>This can be deleted. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3926/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>